README.md

```diff
@@ -74,6 +77,33 @@ To include this project as a dependency:
 grpcio = "0.1"
 ```
 
+Option 2 - Programmatic Generation
+----------------------------------
+
+Programmatic generation can be used to generate Rust modules from proto files
```

Comment:
Can we keep these in the README of `protoc-grpcio` and just add a reference here? So changes in `protoc-grpcio` doesn't result in the changes in this file.

Reply:
Sounds good, I updated this branch to reference the `README` instead of including the examples here.